src/features/form/containers/GroupContainer.tsx

```diff
@@ -233,58 +237,71 @@ export function GroupContainer({ id, container, components }: IGroupProps): JSX.
           filteredIndexes={filteredIndexList}
         />
       )}
-      <Grid
-        container={true}
-        justifyContent='flex-end'
-      />
       {edit?.mode !== 'showAll' &&
         edit?.addButton !== false &&
         editIndex < 0 &&
         repeatingGroupIndex + 1 < (container.maxCount === undefined ? -99 : container.maxCount) &&
         addButton()}
       {editIndex >= 0 && edit?.mode === 'hideTable' && (
-        <RepeatingGroupsEditContainer
-          container={container}
-          editIndex={editIndex}
-          setEditIndex={setEditIndex}
-          repeatingGroupIndex={repeatingGroupIndex}
-          id={id}
-          language={language}
-          textResources={textResources}
-          layout={layout}
-          repeatingGroupDeepCopyComponents={repeatingGroupDeepCopyComponents}
-          multiPageIndex={multiPageIndex}
-          setMultiPageIndex={setMultiPageIndex}
-          filteredIndexes={filteredIndexList}
-        />
+        <ConditionalWrapper
+          condition={!isNested}
+          wrapper={(children) => <FullWidthWrapper>{children}</FullWidthWrapper>}
+        >
+          <RepeatingGroupsEditContainer
+            container={container}
+            editIndex={editIndex}
+            setEditIndex={setEditIndex}
+            repeatingGroupIndex={repeatingGroupIndex}
+            id={id}
+            language={language}
+            textResources={textResources}
+            layout={layout}
+            repeatingGroupDeepCopyComponents={repeatingGroupDeepCopyComponents}
+            multiPageIndex={multiPageIndex}
+            setMultiPageIndex={setMultiPageIndex}
+            filteredIndexes={filteredIndexList}
+          />
+        </ConditionalWrapper>
       )}
-      {edit?.mode === 'showAll' &&
+      {edit?.mode === 'showAll' && (
         // Generate array of length repeatingGroupIndex and iterate over indexes
-        Array(repeatingGroupIndex + 1)
-          .fill(0)
-          .map((_, index) => {
-            if (filteredIndexList && filteredIndexList.length > 0 && !filteredIndexList.includes(index)) {
-              return null;
-            }
+        <ConditionalWrapper
+          condition={!isNested}
+          wrapper={(children) => <FullWidthWrapper>{children}</FullWidthWrapper>}
+        >
+          <>
```

Comment:
This empty tag seems pretty useless. There is no need for a fragment here as long as a parent element exists.